quickwit/quickwit-cli/src/index.rs

```diff
@@ -1161,6 +1161,7 @@ mod test {
         Ok(())
     }
 
+    #[test]
```

Comment:
Remove this line.

Reply:
Ooh yeah, I missed this... should be good now